support/ci/deploy_website.sh

```diff
@@ -7,16 +7,9 @@ if [[ "${TRAVIS_PULL_REQUEST}" = "false" ]] && [[ "${TRAVIS_BRANCH}" = "master"
   cd www
   make deploy
 else
-  if ! [[ "${TRAVIS_PULL_REQUEST}" = "false" ]] && [[ "${TRAVIS_BRANCH}" = "master" ]]; then
+  if [[ "${TRAVIS_PULL_REQUEST}" = "false" ]] && [[ "${TRAVIS_BRANCH}" =~ ^acceptance_deploy ]]; then
```

Comment:
I dont _think_ we should need to, effectively what these two changes do are mimic the post-merge travis jobs that we fire off with the sentinels. That branch gets created dynamically by the sentinels and pushed to travis when a user pokes the sentinels with `@thesentinels acceptance` on an open PR.